apps/ui/src/components/Modal/Vote.vue

```diff
@@ -29,9 +29,13 @@ const {
   fetch: fetchVotingPower,
   reset: resetVotingPower
 } = useVotingPower();
+const { votes } = useAccount();
 
 const loading = ref(false);
-const form = ref<Record<string, string>>({ reason: '' });
+const form = ref<Record<string, string>>({
```

Comment:
I think I’d personally set it when open flag changes, because we usually reset data there as Modals can be mounted one time for multiple uses (don’t think that’s the case there though)